### DISCUSSION TITLE
v7.3.6 Release of PowerShell

### DISCUSSION THREAD

**Discussion by adityapatwardhan** [PowerShell/PowerShell#19966] [General]
### Build and Packaging Improvements

<details>

<summary>

<p>Bump .NET to 7.0.306 </p>

</summary>

<ul>
<li>Update Notices file</li>
<li>Don't publish notice on failure because it prevents retry</li>
<li>Bump .NET to 7.0.306 (#19945)</li>
<li>Remove the property disabling optimization (#19952)</li>
<li>Add ProductCode in registry for MSI install (#19951)</li>
<li>Update variable used to bypass the blocking check for multiple NuGet feeds (#19953)</li>
<li>Change <code>System.Security.AccessControl</code> preview version to stable version (#19931)</li>
</ul>

</details>

### Documentation and Help Content

- Update the link for getting started in `README.md` (#19947)

[7.3.6]: https://github.com/PowerShell/PowerShell/compare/v7.3.5...v7.3.6

### SHA256 Hashes of the release artifacts

- hashes.sha256
  - E0069245F2D02AF88EB8A9905031CE2B48B78FEC3ECBFCD55C66ECF3B7E04206
- powershell_7.3.6-1.deb_amd64.deb
  - 9219BC276DA261D6FD1BCE5DCBF898466E70682EA2F17EABFF85841FBE6C9E57
- powershell-7.3.6-1.cm.x86_64.rpm
  - 27A66C309B6043FA755B56B2AFE5E2A729B50A97DCAC341B13A34EC19EB6CF86
- powershell-7.3.6-1.rh.x86_64.rpm
  - 61CFA395DA404345F3D47EDA781B70DCA999DEA9DDA0F189B43188FACDA6F1D4
- powershell-7.3.6-linux-alpine-x64.tar.gz
  - E059B94B99011D37DE77C07CD10FBC5BF24FAC67184DBEB80CF6C9057C0BFB5E
- powershell-7.3.6-linux-arm32.tar.gz
  - 0BD7759895546AC04ED81368C7818036FC8B2D9826E476F0F249DAB51A30797B
- powershell-7.3.6-linux-arm64.tar.gz
  - 02702C1373012CE9BD0F6A291D8D2763186CF6A9244CE4D22650BF4A4BD722C3
- powershell-7.3.6-linux-x64.tar.gz
  - 38BFBA9AA4C914ADF9F9A64B7F5832F8533305B696B5F03D0C3D56F9A2D8A5DC
- powershell-7.3.6-linux-x64-fxdependent.tar.gz
  - B98E77AACBE269EF143E97BB2E7F272851D737FEF23E31A5BC3E65CA581BF9BE
- powershell-7.3.6-osx-arm64.pkg
  - 36F796FDE211623A966ED470D1A067EF22731AA1515980B61C0F068A4915D1FD
- powershell-7.3.6-osx-arm64.tar.gz
  - 7199D668A59B5ABFD11EEBA7C2486A19E31A34C00DA3B51D61ADFDB4E5ACB2AF
- powershell-7.3.6-osx-x64.pkg
  - 5D023857BBF987698C053E5E64D9CC78DB3E5EF35D62564767756527DE52D878
- powershell-7.3.6-osx-x64.tar.gz
  - E6FC23EEF2B76A2A3A57FF5CAC56953B4D35E1206B04DC76C954CAEB25D39069
- PowerShell-7.3.6-win-arm32.zip
  - 7AA760FC3C9E0B8F8BFCA408B69BF21491D00CF52ABA0B8905D8C9846F7B0457
- PowerShell-7.3.6-win-arm64.zip
  - 20BA733F54F3141616AE290873081DC5684D92AB5DDBA433BF68DB74F7D47026
- PowerShell-7.3.6-win-fxdependent.zip
  - F89322DB7324D3A237716014F8791440F6223594031FC07CED5BC7CCDF5E6154
- PowerShell-7.3.6-win-fxdependentWinDesktop.zip
  - 60E056E54555858B27AD2677E8186F6DE83E211CD26C224D781BD3DB16E36F18
- PowerShell-7.3.6-win-x64.msi
  - 758E130DC13708A72CEE29DB0B8EFD987376A4A4A22114C3B00D50CF7295D35E
- PowerShell-7.3.6-win-x64.zip
  - FCBD1699BEAED9E42D8D335C60F17C746055C4F05D4D658D2F54B88F99BD01BA
- PowerShell-7.3.6-win-x86.msi
  - 0D295F1AE6B6F0A5531AC33DF0A98126175312FC85252C77738D38CF28AA18EE
- PowerShell-7.3.6-win-x86.zip
  - 1DDAA161D1A590211E2A8E31442F03643D27A2A5BFBB03711669CF047276D13D

<hr /><em>This discussion was created from the release <a href='https://github.com/PowerShell/PowerShell/releases/tag/v7.3.6'>v7.3.6 Release of PowerShell</a>.</em>

**Comment by Nate-Lowery-Lumen** [comment]
When will this be available in winget?

![image](https://github.com/PowerShell/PowerShell/assets/89099065/65e85782-556c-435f-accc-cbdff9cbad02)

**Reply by kingabzpro** [reply]
Same.

**Reply by kilasuit** [reply]
We were waiting for [this PR](https://github.com/microsoft/winget-pkgs/pull/112154) to be merged for this to be available in Winget package feed when you raised this comment

**Reply by kilasuit** [reply]
It should now be available though

**Comment by wilburlikesmith** [comment]
![image](https://github.com/PowerShell/PowerShell/assets/1489940/0435a87a-b1d7-4b25-b1b1-a5231ce48473)

Same. 

```
PowerShell 7.3.5

    A new PowerShell stable release is available: v7.3.6
    Upgrade now, or check out the release page at:
    https://aka.ms/PowerShell-Release?tag=v7.3.6

PS C:\> winget upgrade Microsoft.Powershell --version 7.3.6

    No version found matching: 7.3.6

PS C:\> winget search microsoft.powershell

    Name               Id                           Version Source
    ---------------------------------------------------------------
    PowerShell         Microsoft.PowerShell         7.3.5.0 winget
    PowerShell Preview Microsoft.PowerShell.Preview 7.4.0.4 winget

PS C:\>
```

**Reply by kilasuit** [reply]
We were waiting for [this PR](https://github.com/microsoft/winget-pkgs/pull/112154) to be merged for this to be available in Winget package feed when you raised this comment

It should be available now though

**Comment by JustinGrote** [comment]
@adityapatwardhan the build and packaging improvements don't quite collapse correctly at least on Windows. I think the "bump" just needs to have its own list radio and then it should be fine
![image](https://github.com/PowerShell/PowerShell/assets/15258962/4344c424-6ef4-4072-b8e2-b593ac7a01b1)

**Reply by adityapatwardhan** [reply]
I can have a look at that.

**Comment by monstertsl** [comment]
![image](https://github.com/PowerShell/PowerShell/assets/39371736/a6aaef94-5ff7-4f91-a5c1-b3a5f8108964)

**Comment by JustinGrote** [comment]
@monstertsl what is your `$env:PATH` look like? Works fine for me.

Also I see you're using a custom prompt and therefore I assume a custom profile, have you tried `pwsh -noprofile`?
